src/python/ddapp/tabledemo.py

```diff
@@ -667,6 +720,9 @@ def playSequenceNominal(self):
         self.planPlaybackFunction(self.plans)
 
     def commitManipPlan(self):
+        if self.planner == 1:
```

Comment:
Perhaps switch using visOnly or expose visOnly in GUI rather than couple playback/real commit to planner